patterns/services-three-column-boxed-pricing-table.php

```diff
@@ -0,0 +1,112 @@
+<?php
+/**
+ * Title: Services - Three-column boxed pricing table
+ * Slug: twentytwentyfive/services-three-column-boxed-pricing-table
+ * Categories: call-to-action, banner, featured, services
+ * Description: A three-column boxed pricing table designed to showcase services, descriptions, and pricing options.
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"var:preset|spacing|60","bottom":"var:preset|spacing|60","left":"var:preset|spacing|50","right":"var:preset|spacing|50"},"blockGap":"var:preset|spacing|50","margin":{"top":"0","bottom":"0"}}},"layout":{"type":"constrained"}} -->
```

Comment:
The blockGap / block spacing is meant to be on the columns block.
To adjust the space between the columns.

Reply:
@carolinan Apologies, I couldn't understand this earlier, but I have now resolved the issue.